src/config/Theme/index.ts

```diff
@@ -0,0 +1,14 @@
+import { getMuiThemeConfig } from './muiThemeConfig';
+import { customTheme as rainbowTheme } from './rainbowTheme';
+import { lightTheme, darkTheme } from './theme';
+
+export const getCustomThemes = () => {
+  return {
+    main: {
+      light: lightTheme,
+      dark: darkTheme,
+    },
+    mui: getMuiThemeConfig,
```

Comment:
If it's a method, I'd prefer to keep the get prefix:

```suggestion
    getMui: getMuiThemeConfig,
```

Reply:
fixed: [051c63d](https://github.com/defi-wonderland/web3-react-boilerplate/pull/21/commits/051c63d0c3dce26ec23203f015deabf3b3adf64e)